nixos/modules/misc/documentation.nix

```
@@ -353,8 +384,17 @@ in
         '';
       };
 
-    };
+      nixpkgs.enable = mkOption {
+        type = types.bool;
+        default = true;
```

```suggestion
        default = false;
```
I'm not sure this should be enabled by default. Does every NixOS user need the Nixpkgs documentation? I think it's best to leave this off by default (`mkEnableOption`) and add a note on the Contribution Guide or the Nixpkgs documentation about the existence of this option.

---

Oops, this was supposed to default to `false`.